tests/test_wave_03.py

```diff
@@ -59,7 +59,7 @@ def test_friends_unique_movies_not_duplicated():
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
```

Comment:
The `assert` that came with the test lets us know how many movies to expect in the list `friends_unique_movies`, what can we assert to make sure the data each movie holds is correct?